megamek/data/images/units/mechset.txt

```diff
@@ -5946,106 +5946,106 @@ chassis "Yurei_airmech" "mechs/Yurei LAM_AirMech.png"
 chassis "Yurei_fighter" "mechs/Yurei LAM_Fighter.png"
 
 ###DROPSHIPS
-chassis "Aesir Assault Dropship" "dropships/Aesir.png" 
-chassis "Aqueduct Liquid Carrier" "dropships/Aqueduct.png" 
-chassis "Arcadia" "dropships/Arcadia.png" 
+chassis "Aesir Assault Dropship" "dropships/Aesir.png"
```

Comment:
Are all of these non-turret changes expected?  Were these related to the turret changes, or brought in from other changes?